### PR TITLE
feature: set hidden overflow text to display an ellipsis

### PR DESCRIPTION
**Pull request by LiKang6688**
Learning from the old straight table

Screenshot 2021-05-04 at 09 22 30

### CODE REVIEW

File: src/table/TableHeadWrapper.jsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default function TableHeadWrapper({
                 }
                 tabIndex={-1}
               >
-                {column.label}
+                <div>{column.label}</div>
```
Comment (haxxmaxx) [inline src/table/TableHeadWrapper.jsx:67]: Hmm not sure about this. How does it affect the screen reader?

File: src/mui-setup.js [hunk 2]
```diff
@@ -16,6 +16,25 @@ export default function muiSetup(sproutBase) {
       display: 'table',
       width: '100%',
       tableLayout: 'fixed',
+      ...sproutBase.overrides.MuiTableRow.root,
+    };
+
+    const cellStyle = {
+      maxHeight: '3.7em',
```
Comment (haxxmaxx) [inline src/mui-setup.js:23]: why 3.7?

Comment (LiKang6688) [reply src/mui-setup.js:23]: basically at least to show two rows in the table

File: src/table/TableHeadWrapper.jsx (hunk 1 above)
Comment (LiKang6688) [reply src/table/TableHeadWrapper.jsx:67]: Screenshot 2021-05-04 at 10 21 55
looks good in sense-client

Comment (LiKang6688) [reply src/table/TableHeadWrapper.jsx:67]: Screenshot 2021-05-04 at 10 25 52
looks good in sense-client

**Comment by haxxmaxx** [comment]
Tested a bit and it doesn't look great. I never get a horizontal scroll bar, so I guess there has to be some kind of min width?

**Comment by LiKang6688** [comment]
> Tested a bit and it doesn't look great. I never get a horizontal scroll bar, so I guess there has to be some kind of min width?

no, it is a side effect from https://github.com/qlik-oss/sn-table/pull/184. Looking into solutions now.

**Comment by LiKang6688** [comment]
> Tested a bit and it doesn't look great. I never get a horizontal scroll bar, so I guess there has to be some kind of min width?

it is an issue when I

**Pull request closed without merging**